code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -362,7 +362,7 @@
 	H.visible_message("<b>[H]</b> falls to the ground and screams as their skin bubbles and froths!") //'froths' sounds painful when used with SKIN.
 	H.Weaken(3)
 	sleep(30)
-	var/list/blacklisted_species = list(/datum/species/zombie, /datum/species/skeleton, /datum/species/human, /datum/species/golem, /datum/species/golem/adamantine, /datum/species/shadow, /datum/species/shadow/ling, /datum/species/plasmaman)
+	var/list/blacklisted_species = list(/datum/species/zombie, /datum/species/skeleton, /datum/species/golem, /datum/species/golem/adamantine, /datum/species/shadow, /datum/species/shadow/ling, /datum/species/shadow/ling/lesser, /datum/species/plasmaman, /datum/species/plasmaman/skin)
```

Comment:
Dangerous_existence is relative to the person who has the race, it doesn't mean the race is inherently more dangerous <i>to others</i>

When synths were added to that list I was confused for much the same same reason, because being a synth isn't actually an occupational hazard.

Maybe there should be two lists.